mysql-test/suite/rocksdb/t/mysqldump.test

```diff
@@ -34,30 +34,31 @@ insert into r1 values (2,2,2,1,15,15,15,15);
 insert into r1 values (2,2,2,2,16,16,16,16);
 
 connection con2;
+SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED;
 BEGIN;
 insert into r1 values (5,5,5,5,5,5,5,5);
 update r1 set value1=value1+100 where id1=1 and id2=1 and id3='1';
 
---exec $MYSQL_DUMP --skip-comments --single-transaction --master-data=2 --print-ordering-key --rocksdb --order-by-primary-desc --rocksdb_bulk_load test
```

Comment:
--print-ordering-key was a Facebook MySQL addition to mysqldump that we have not/did not take  in and is not related to MyRocks. It came in this commit to Facebook MySQL 2+ years ago https://github.com/facebook/mysql-5.6/commit/0f2578a086ce7822ce16fb9ac5c47e515ea2b1c5

It is what generated the "/* ORDERING KEY (DESC) : PRIMARY */;" output above and came in the MyRocks test results during the initial import. Now that we are re-recording these MyRocks tests to actually pass with our implementation of consistent snapshot and other bits of this pull request, these are disappearing.